be/src/storage/lake/persistent_index_sstable.cpp

```diff
@@ -49,8 +50,10 @@ Status PersistentIndexSstable::build_sstable(
     for (const auto& [k, v] : map) {
         IndexValueWithVerPB index_value_pb;
         for (const auto& index_value_with_ver : v) {
-            index_value_pb.add_versions(index_value_with_ver.first);
-            index_value_pb.add_values(index_value_with_ver.second.get_value());
+            auto* item = index_value_pb.add_items();
+            item->set_version(index_value_with_ver.first);
```

Comment:
In `types.proto`, the comments said that verison will be set if MVCC is supported. So should version be set now?

Reply:
will removed in next pr